### ISSUE TITLE
[BUG] Client crashes upon using a Honeycomb on a Copper golem due to a mod conflict

### ISSUE THREAD

**Issue by SplendidAlakey** [How-Bout-No/Outvoted#96]
/Before submitting, please ensure you've done the following, as these are fixes for some common bugs:
[✔] Reset/delete config files
[✔] Run without mods like Optifine (if needed)
[✔] <ins>Double check that you have the latest version!!</ins>

**Describe the bug**

When having a lot of mods loaded together with Outvoted, the client crashes upon using a Honeycomb on a Copper golem when playing on a server.

With all my mods I consistently get a crash after using a Honeycomb on a Copper golem, with just Outvoted everything works as intended.

Only the client crashes, server keeps working. In single player no crash happens with the same exact mods. I also don't see any related logs in the latest.log of the server.

**To Reproduce**

1. Have all the mods I do on both the client and the server
2. Join the server and try to use a Honeycomb on a Copper golem
3. Client will crash, server will not

**Screenshots/Logs**

[removed]

**Version (please complete the following information):**

 - Minecraft Version 1.17.1
 - Modloader Fabric 0.12.3, FAPI 0.41.0
 - Mod Version 1.5.0-b1

**Additional context**

My modpack is pretty big so going through every mod 1 by 1 would be very time consuming, so I'm hoping these logs could help identify the issue.

**Comment by How-Bout-No** [comment]
I'll see if I can narrow down the issue. Seeing as this is a client only crash, it should be easier to deduce.

**Comment by How-Bout-No** [comment]
Also is this a modpack on CurseForge or is this one you setup? Cause it would be easier for me to pick apart that way

**Comment by SplendidAlakey** [comment]
It's my own custom modpack. I based it on [Better Minecraft](https://www.curseforge.com/minecraft/modpacks/better-minecraft-fabric) and [All Of Fabric 4](https://www.curseforge.com/minecraft/modpacks/all-of-fabric-4), but I have some mods they don't and vice versa.

**Comment by How-Bout-No** [comment]
Would you be able to test this jar? Considering this is a client crash it should probably only be needed on the client (assuming it works :P) [outvoted-1.5.0-b2.zip](https://github.com/How-Bout-No/Outvoted/files/7413646/outvoted-1.5.0-b2.zip)

**Comment by SplendidAlakey** [comment]
It works. The crash doesn't happen anymore. Thank you!

I noticed that now they can only be waxed once (could be waxed infinitely before), which I would guess was the issue?

**Comment by How-Bout-No** [comment]
Actually, that's a separate bug that I also fixed in this dev version lol. I believe the problem was that code for the axe interaction was accidentally used for the Honeycomb (and copper ingot too) and so it was trying to damage the Honeycomb item when it couldn't? Theoretically this should also probably crash the server too but maybe I just don't get it. I also went and removed some ambiguity between clients and servers in some separate code which seems to have no side effect but this I'll need to test myself.
